### DISCUSSION TITLE
3ds port is dope but...

### DISCUSSION THREAD

**Discussion by Joenzii** [nzp-team/nzportable#407] [General]
Would love to see some more controller configurations added unless im missing them would level to be able to bind reload and knife to diffrent weapons and such. Keep putting love into this I love it

**Comment by pahntomforces-guy** [comment]
i love this game, i played it when i was 8 yrs old :D

**Comment by NZPUSER** [comment]
where would i get the download for my new 3ds xl cause i wanna play on it 
if you could reply that would be great thanks

sincerely a new 3ds xl owner who want this game

**Reply by MotoLegacy** [reply]
The same place you get it everywhere else... https://github.com/nzp-team/nzportable/releases/latest

Also available from universal updater.
